package.json

```diff
@@ -41,10 +41,11 @@
     "redux-persist": "^5.10.0",
     "redux-saga": "^0.16.1",
     "stacktrace-js": "^2.0.0",
-    "typesafe-actions": "^2.0.4",
+    "typesafe-actions": "3.4.0",
```

Comment:
Okay, updating some of our source types.

Also updating the base tslint to a shared repo to reuse across the projects.